pkg/strategy/xgap/strategy.go

```diff
@@ -355,9 +358,11 @@ func (s *Strategy) CrossRun(ctx context.Context, _ bbgo.OrderExecutionRouter, se
 					log.WithError(err).Error("order submit error")
 				}
 
+				s.activeOrderBook.Add(createdOrders...)
+
 				time.Sleep(time.Second)
 
-				if err := tradingSession.Exchange.CancelOrders(ctx, createdOrders...); err != nil {
+				if err := tradingSession.Exchange.CancelOrders(ctx, s.activeOrderBook.Orders()...); err != nil {
```

Comment:
Can we use s.activeOrderBook.GracefulCancel(ctx) here ?

Reply:
ok~